kai/rpc_server/server.py

```diff
@@ -282,6 +282,15 @@ def initialize(
             ],
         )
 
+        app.log.info("Priming analyzer cache with default run")
+
+        app.analyzer.run_analyzer_lsp(
```

Comment:
I wonder if this can be fired and returned so we are ready to work? 

I guess the question is, how much time does this add to start up, and do we want to "do" something different